include/internal/benchmark/catch_constructor.hpp

```diff
@@ -64,7 +62,7 @@ namespace Catch {
                 }
 
 
-                TStorage data;
+                struct { alignas(T) unsigned char data[sizeof(T)]; }  data;
```

Comment:
Why the anonymous struct?

Reply:
That's the expansion of `aligned_storage::type`, basically. I presume it's to avoid the result from decaying to a `uchar*`: https://en.cppreference.com/w/cpp/types/aligned_storage#Possible_implementation

I'll update the commit message.

Reply:
Hmmm, makes sense I guess.